main.c

```diff
@@ -169,7 +207,7 @@ void executeProfile(){
  * Turn off all leds
  */
 void disableLeds(){
-  currentProfile = (currentProfile+amountOfProfiles-1)%amountOfProfiles;
+  // currentProfile = (currentProfile+amountOfProfiles-1)%amountOfProfiles;
```

Comment:
please just remove this line altogether. Tying to not have a lot of "commented out" code.